packages/edit-site/src/components/editor-canvas-container/index.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * WordPress dependencies
+ */
+import { Children, cloneElement, useState, useMemo } from '@wordpress/element';
+import { createSlotFill, Button } from '@wordpress/components';
+import { ESCAPE } from '@wordpress/keycodes';
+import { __ } from '@wordpress/i18n';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { closeSmall } from '@wordpress/icons';
+import { useFocusOnMount, useFocusReturn } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../private-apis';
+import { store as editSiteStore } from '../../store';
+
+/**
+ * Returns a translated string for the title of the editor canvas container.
+ *
+ * @param {string} view Editor canvas container view.
+ *
+ * @return {string} Translated string corresponding to value of view. Default is ''.
+ */
+export function getEditorCanvasContainerTitle( view ) {
+	switch ( view ) {
+		case 'style-book':
+			return __( 'Style Book' );
+		default:
+			return '';
+	}
+}
+
+const SLOT_FILL_NAME = 'EditSiteEditorCanvasContainerSlot';
+const { Slot: EditorCanvasContainerSlot, Fill: EditorCanvasContainerFill } =
+	createSlotFill( SLOT_FILL_NAME );
```

Comment:
I'd recommend making it a private slot/fill if it's an internal implementation detail.

`createPrivateSlotFill` was added in https://github.com/WordPress/gutenberg/pull/49819.

Reply:
Good call. Done in [b2a2a70](https://github.com/WordPress/gutenberg/commit/b2a2a70ff643fa3a8a32d144c9f37d2cb2e1214e)